YandexMoneyCoreApi/Core/ApiMethod.swift

```diff
@@ -22,6 +22,7 @@
  */
 
 import struct Foundation.URL
+import Foundation
```

Comment:
You must append specific import or remove `import struct Foundation.URL`.
The first option is preferable.